reference/checkouts.sf.yml

```diff
@@ -2563,6 +2568,9 @@ definitions:
           id:
             description: ''
             type: string
+          shouldSaveAddress:
+            description: 'Indicates if we should add this address to the customer address book'
```

Comment:
'Indicates if we should add this address to the customer address book.'

Reply:
Thanks, I updated the description